### DISCUSSION TITLE
Use of tern:Attribute

### DISCUSSION THREAD

**Discussion by miekeGR** [ternaustralia/ontology_tern#104] [Q&A] [answered]
Can I add a tern:Attribute to an Observation? I want to an observation that hasResult tern:Text with a category that is more specific than the observedProperty gives.

**Comment by edmondchuc** [comment, marked as answer]
Please see its usage in the cookbook at https://linkeddata.tern.org.au/information-models/tern-ontology/cookbook/annotating-instances-with-the-attribute-class.

**Reply by miekeGR** [reply]
Cheers, thank for the pointer @edmondchuc
